gh-badges/lib/make-badge.spec.js

```diff
@@ -139,6 +139,48 @@ describe('The badge generator', function() {
     })
   })
 
+  describe('"flat" template badge generation', function() {
+    it('should match snapshot', function() {
```

Comment:
I'm also in 2 minds about these tests. I added them in fcf6c68 because I thought they would help but they now essentially duplicate one case of the more extensive tests that exist in `gh-badges/lib/badge-renderers.spec.js`, but one layer further out in the application. They're not doing any harm, but I'm not totally convinced its useful to duplicate them. Again, any thoughts?

Reply:
> I'm also in 2 minds about these tests

Just to ensure I'm looking at the right set of tests, how many of the below tests fall into this category for you? Just down to line 182 or all the `it('should match snapshot', ...)` tests?

Reply:
> how many of the below tests fall into this category for you?

All the ones added in fcf6c68

> my preference would be to test it at this layer rather than one level down

Yeah that makes sense. That kind of answers the other question too.

Reply:
62529b2